pkg/internal/controller/controller.go

```diff
@@ -264,6 +264,7 @@ func (c *Controller) reconcileHandler(ctx context.Context, obj interface{}) {
 		c.Queue.AddRateLimited(req)
 		ctrlmetrics.ReconcileErrors.WithLabelValues(c.Name).Inc()
 		ctrlmetrics.ReconcileTotal.WithLabelValues(c.Name, "error").Inc()
+		c.Log.Error(err, "Reconciler error")
```

Comment:
```suggestion
		log.Error(err, "Reconciler error")
```
So we get name and namespace information as well